test/metabase/driver/test_util.clj

```diff
@@ -0,0 +1,18 @@
+(ns metabase.driver.test-util
+  (:require
+   [metabase.driver :as driver]
+   [metabase.test.initialize :as initialize]))
+
+(defn notify-all-databases-updated []
+  ;; It makes sense to notify databases only if app db is initialized.
+  (when (initialize/initialized? :db)
+    (initialize/initialize-if-needed! :plugins)
+    (#'driver/notify-all-databases-updated)))
+
+(defmacro wrap-notify-all-databases-updated
+  [& body]
+  `(do (notify-all-databases-updated)
```

Comment:
Is this really needed?

Reply:
If _this_ refers to `do` block, I could theoretically put `notify-all-databases-updated` in the `try` block. But then, if it throws, it would be called again in `finally`. And that's something I'd like to avoid. Please let me know if there is some error in my reasoning. Thanks.

Reply:
This refers to the `(notify-all-databases-updated)` call. Are there two such calls because we don't trust tests clean up after themselves properly?

Reply:
I believe `mt/with-report-timezone-id` should do the clean up when done. See the change to `do-with-report-timezone-id`.